certbot/certbot/_internal/cli/__init__.py

```diff
@@ -309,6 +309,22 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):
     helpful.add(
         "security", "--rsa-key-size", type=int, metavar="N",
         default=flag_default("rsa_key_size"), help=config_help("rsa_key_size"))
+    helpful.add(
+        "security", "--elliptic-curve", type=str, choices=[
```

Comment:
Note to self, this should only be a list of three curves.

Reply:
I think it is the case now.

Reply:
Yes. It wasn't when I wrote the comment. I'll mark it as resolved :)